beacon_node/http_api/src/lib.rs

```diff
@@ -3869,12 +3869,16 @@ pub fn serve<T: BeaconChainTypes>(
                         )));
                     }
 
-                    let liveness: Vec<api_types::StandardLivenessResponseData> = indices
+                    let liveness: Vec<api_types::LivenessResponseData> = indices
                         .iter()
                         .cloned()
                         .map(|index| {
                             let is_live = chain.validator_seen_at_epoch(index as usize, epoch);
-                            api_types::StandardLivenessResponseData { index, is_live }
+                            api_types::LivenessResponseData {
```

Comment:
I don't think we are looking to change the response of this endpoint as it will break spec compliance. 

https://ethereum.github.io/beacon-APIs/?urls.primaryName=v2.4.0#/Validator/getLiveness